rrtmgp/kernels-openacc/mo_gas_optics_kernels.F90

```diff
@@ -402,10 +402,81 @@ subroutine gas_optical_depths_minor(ncol,nlay,ngpt,        &
     integer  :: icol, ilay, iflav, igpt, imnr
     integer  :: gptS, gptE
     integer  :: minor_start, minor_loc, extent
+
+    real(wp),    dimension(ncol,nlay,ngpt) :: tau_t
```

Comment:
It looks like `tau_t` isn't ever used again?

Reply:
Whoops! Removed